setup.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (C) 2019, RS
+# Full license can be found in License.md
+# -----------------------------------------------------------------------------
+
+import os
+from codecs import open
```

Comment:
Recommend not renaming existing function, `open`.

Reply:
Code updated.